packages/handlersjs-core/lib/daemons/scheduler.ts

```diff
@@ -0,0 +1,59 @@
+import { Observable, Subject } from 'rxjs';
+import { Daemon } from '../models/daemon';
+
+export class Scheduler extends Daemon {
+
+  private currentTimeout: NodeJS.Timeout | undefined;
```

Comment:
Unless it's needed for some conflicting names, prefer to use an import and the unscoped type.